prep-javascript-variables/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+   const fullName = 'Junou Lee';
+   console.log(fullName);
+   const numberOfPets = '2';
```

Comment:
For numbers, we'll want to leave off the quotes, so that the JavaScript engine doesn't interpret it as a string.

Reply:
![image](https://user-images.githubusercontent.com/117887912/203378187-b100d31e-1938-42de-9200-52eee8a769dc.png)

Reply:
![image](https://user-images.githubusercontent.com/117887912/203378260-58f49025-2f17-46bd-9df5-77bb909ca65d.png)